frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -105,6 +105,18 @@ export class StorageClassDropdownInner extends React.Component<
     }
   }
 
+  getLabelClassName = () => {
+    const classNames: string[] = [];
+    this.props.hideClassName
+      ? classNames.push(`${this.props.hideClassName} control-label`)
+      : classNames.push('control-label');
+
+    if (this.props.required) {
+      classNames.push('co-required');
+    }
+    return classNames.join(' ');
```

Comment:
Use the `classNames` utility, which is already a dependency.

https://github.com/JedWatson/classnames

You can probably just inline it in `render` using `classNames`:

```tsx
const labelClassName = classNames('control-label', this.props.hideClassName, { 'co-required': this.props.required });
```

Reply:
i've changed to use this